src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -29,7 +32,30 @@
 
 logger = _logging.get_az_logger(__name__)
 
-def dcos_browse(name, resource_group_name):
+def wait_then_open(url):
+    """
+    Waits for a bit then opens a URL.  Useful for waiting for a proxy to come up, and then open the URL.
+    """
+    # TODO: we should ping the URL instead of just sleeping.
+    time.sleep(2)
+    webbrowser.open_new_tab(url)
+
+def wait_then_open_async(url):
+    t = threading.Thread(target=wait_then_open, args=({url}))
+    t.daemon = True
+    t.start()
+
+def k8s_browse(disable_browser=False):
+    """
+    Wrapper on the 'kubectl proxy' command, for consistency with 'az dcos brows'
+    """
+    logger.info('Proxy running on 127.0.0.1:8001/ui')
```

Comment:
This should be `logger.warning` I believe.
`logger.info` will only be should to the user if they add the `--verbose` or `--debug` flags.